tests/integration/targets/azure_rm_virtualnetworkgatewayconnection/tasks/main.yml

```diff
@@ -55,48 +55,14 @@
       common: "xyz"
   register: network_gateway1_output
 
-- name: Create virtual network
```

Comment:
Can we keep these tests for regression test? Instead we shall incrementally add new tests for new coverage. Also, for this specific one, can we make these two sets of vnet gateway to ceate in parallel?